resident-services/README.md

```diff
@@ -176,4 +189,27 @@
 
 * Channel Verification Status (Preparation) - This thread contains two APIs i.e Request OTP and Validate OTP which are added in a loop controller because we are running both the APIs for otp channel type as "EMAIL" and "PHONE". So will save the individual id and type of channel in a file named as channel_verification_list.txt. Loop count for the loop controller is defined with the variable name "otpChannelTypeLoopCount" which is currenlty set to 2. If required, we can add the channel type to the file and as per the count of otp channel type we can modify the variable defined in the script.
 
-* Channel Verification Status (Execution) - This API is to check if OTP is verified for a channel type for an individual VID. The individual id and channel type will be passed in this API from a file named as channel_verification_list.txt and the variable used are "individualIdVerification" and "channelVerification".
+* Channel Verification Status (Execution) - This API is to check if OTP is verified for a channel type for an individual VID. The individual id and channel type will be passed in this API from a file named as channel_verification_list.txt and the variable used are "individualIdVerification" and "channelVerification". 
+
+* Auth Type Callback (Execution) - This API is a Websub callback for Auth Type Lock/Unlock acknowledgement from IDA. Resident Service subscribes to the websub topic AUTH_TYPE_STATUS_UPDATE_ACK and update the Auth type lock/unlock status into resident_transaction. A BeanShell PreProcessor is used to generate SHA256 value using the request body of the API and secret key 
+of resident-websub-authtype-status-secret which is passed in the headers.
+
+* Download Card Event (Preparation) - In this thread we are using Update UIN API from which will get an Event id which is stored in a file named as event_id_download_card.txt. The id and access token will be used here and can be used multiple times untill its valid and not expired.
```

Comment:
If this is only about the downloading the card, can't we use the request to generation of card as prepration since it will little more light weight comparing a uin update where the packet needs to be processed from regproc

Reply:
We have changed the preparation API. We are now using Request Card VID API from which will get the event id.